packages/react-sdk/src/useDisplayName.ts

```diff
@@ -0,0 +1,27 @@
+import { useMemo } from 'react'
+import { useSyncAgent } from './useSyncAgent'
+import { useObservable } from './useObservable'
+import { useAction } from './internals/useAction'
+
+export const useDisplayName = (spaceId: string, userId: string) => {
```

Comment:
I'm not handling per channel metadata, since we need a way to know if the dev wants the metadata from the channel or the metadata from the space

I'm thinking about the following:
```ts
useDisplayName(userId, spaceId, channelId, { priority: 'channel' | 'space' })
```

Reply:
can it be `useDisplayName(member: Member)` so we don't need to have optional observables? 
i realize like eventually it will probably be necessary, but we can kick it down the road.

Reply:
`useDisplayName(member: Member)` will raise the undefined check to the caller

```ts
const Component = () => {
   const bob = space.members.getMember(bobId)
   const { username } = useUsername(bob) ❌ 
   // bob can be undefined
}
```
```ts
const Component = () => {
   const bob = space.members.getMember(bobId)
   // ...
   return <>{bob && <MemberData member={bob} />}</>
}

const MemberData({ member } : { member: Member }) {
   const { username } = useUsername(member) ✅
   const { displayName } = useDisplayName(member) ✅
  // ...
}
```

I think this works for now, and I like the idea of passing a Member object, but it can cause some mistakes like calling `Member.displayName` directly, but I think it's fine for a v1